Signal/src/Models/OWSMessagesBubblesSizeCalculator.m

```diff
@@ -52,6 +55,7 @@ - (instancetype)init
     if (self = [super init]) {
         _referenceSystemMessageCell = [OWSSystemMessageCell new];
         _referenceUnreadIndicatorCell = [OWSUnreadIndicatorCell new];
+        _referenceContactOffersCell = [OWSContactOffersCell new];
```

Comment:
This new cells works a bit like the "unread indicator" cell. It's full width and doesn't support timestamp headers.